docs/source/roadmap.md

```diff
@@ -36,6 +36,8 @@ The items in this section are the core team's priorities.
 
 ### Next major releases
 
+- Support building mobile apps using CapacitorJS;
```

Comment:
@fredmaiaarantes in my opinion the Cordova integration (mirroring Cordova commands/config) has never worked nicely.

I see the benefit of having both within one project folder, but would not recommend mirroring the commands/config like Meteor did with Cordova.

We are already using Meteor with Capacitor as separate projects, so I'd happily provide any feedback here (or share some examples).

I would personally recommend using the same project structure as Capacitor itself for Android/iOS/Windows: when initializing Capacitor create a capacitor folder within the Meteor project and create a 'sync' command which copies the built Meteor assets to the Capacitor folder.

Reply:
hey @KoenLav, I'd love to hear more about your experience on this. Do you have anything like this 'sync' command already in your projects? If you could share examples that would be really great.

Reply:
We don't have a sync command since we make use of server.url in Capacitor (so no need to sync the assets), which forces us to monitor compatibility between web/native ourselves, but enables us to perform updates simply by executing window.location.reload() (also it enables us to switch a single Tenant to another instance by modifying something in the database; window.location = anothersubdomain.ourapp.com).

You might expect the native functionality not to work when the web assets aren't bundled; but it works just fine as Capacitor injects itself (and any registered plugins) at every pageload (of a whitelisted domain, of course).

I'm not sure what I would recommend for Meteor as a default scenario, as Capacitor docs note using server.url is "only for development", but somewhere they clarified this is only the case because iOS doesn't really allow apps which can update themselves outside the app store (although it seems Apple isn't really being a PITA about it anymore).

You can give using server.url a try within a few minutes, simply run meteor on localhost:3000 and point server.url there (setting server.cleartext = true in Capacitor config as well).

Either way I would be more than happy to shed some light on our process as the only considerable difference between using 'sync' or 'server.url' is "where are the files coming from (web vs storage)", and I have some basic knowledge of how using sync would work since we experimented with that at first as well.

As a basic outline I would expect the following (pseudo):
- meteor capacitor init => meteor npm install @capacitor/core && mkdir ./capacitor && npm install @capacitor/core && npm install @capacitor/cli --save-dev && npx cap init
- meteor capacitor sync => meteor build && mv ./.meteor/local/build/programs/web.browser ./capacitor/www

Note: simply copying the built assets would probably not be sufficient since Capacitor expects an index.html, but that should be relatively easy to whip up.

npx cap init will start the normal Capacitor installation, which I think is just fine if we use sync, but if server.url would be the default we would probably need to overwrite/append to the config file (server block).

That's basically it, but I'd happily schedule a call to go over some details: https://forums.meteor.com/u/koenlav

Reply:
Thanks for sharing, I had shared your comments here and we will reach out to you when we start working on this.